### PR TITLE
Add list of block explorers from octez docs

### PR DESCRIPTION
**Pull request by timothymcmackin**
The Octez docs are removing a page with links to community pages. This PR pulls the list of block explorers into the Tezos docs.

### CODE REVIEW

File: docs/developing/information/block-explorers.md [hunk 1]
```diff
@@ -35,5 +35,11 @@ Block explorers are also used by:
...
-1. [TzStats](https://tzstats.com/)
-2. [TzKT](https://tzkt.io/)
+- [TzStats](https://tzstats.com/)
+- [TzKT](https://tzkt.io/)
+- [Arronax](https://arronax.io)
+- [Baking Bad](https://baking-bad.org)
+- [Better Call Dev](https://better-call.dev)
+- [https://explorus.io/](https://explorus.io/)
+- [Etherlink Explorer](https://explorer.etherlink.com/)
+- [https://tzflow.com/](https://tzflow.com/)
```
Comment (beatalipska) [inline docs/developing/information/block-explorers.md:45]:
```suggestion
- [TzFlow](https://tzflow.com/)
```

File: docs/developing/information/block-explorers.md [hunk 2]
```diff
@@ -35,5 +35,11 @@ Block explorers are also used by:
...
 Here are some commonly used Tezos block explorers:
 
-1. [TzStats](https://tzstats.com/)
-2. [TzKT](https://tzkt.io/)
+- [TzStats](https://tzstats.com/)
+- [TzKT](https://tzkt.io/)
+- [Arronax](https://arronax.io)
+- [Baking Bad](https://baking-bad.org)
+- [Better Call Dev](https://better-call.dev)
+- [https://explorus.io/](https://explorus.io/)
```
Comment (beatalipska) [inline docs/developing/information/block-explorers.md:43]:
```suggestion
- [Explorus](https://explorus.io/)
```

**Review by beatalipska — CHANGES_REQUESTED** [review]
I thought for the consistency, would be good to use Name of the explorer that is linked to the website

File: docs/developing/information/block-explorers.md (hunk 1 above)
Comment (timothymcmackin) [reply docs/developing/information/block-explorers.md:45]: Yes, that's consistent, thanks.

**Pull request merged**